modin/pandas/indexing.py

```diff
@@ -391,12 +391,16 @@ def _broadcast_item(self, row_lookup, col_lookup, item, to_shape):
                     "an iterable"
                 )
             if hasattr(item, "columns"):
-                if not all(idx in item.columns for idx in col_lookup):
+                own_cols = self.qc.columns
+                # convert indices in column lookup to column names, as Pandas reindex expects them so
+                lookup = [own_cols[idx] for idx in col_lookup]
```

Comment:
isn't `lookup = self.qc.columns[col_lookup]` will give us the same result?

Reply:
Indeed, I missed this point, thx